Sources/Misc/Deprecations.swift

```diff
@@ -259,6 +259,15 @@ public extension Purchases {
         self.attribution.setCreative(creative)
     }
 
+    @available(iOS, deprecated, renamed: "purchase(_:completion:)")
+    @available(tvOS, deprecated, renamed: "purchase(_:completion:)")
+    @available(watchOS, deprecated, renamed: "purchase(_:completion:)")
+    @available(macOS, deprecated, renamed: "purchase(_:completion:)")
+    @available(macCatalyst, deprecated, renamed: "purchase(_:completion:)")
```

Comment:
Just a small comment. We could save some lines with 
```
@available(*, deprecated, renamed: "purchase(_:completion:)")
```
But I see that previous deprecations are done the same (explicitly for each platform) so I guess that, for consistency, we should keep doing it that way.